notification/alert.go

```diff
@@ -9,11 +9,17 @@ type Alert struct {
 	Details     string
 	ServiceID   string
 	ServiceName string
+	Meta        []AlertMeta
```

Comment:
The []{key, value} stuff is just for GraphQL because the language does not support any kind of map.

```suggestion
	Meta        alert.Metadata
```